src/ex4/server/helper/logger.js

```diff
@@ -0,0 +1,11 @@
+import { writeToFile, readFile } from './fileHandler.js'
+
+export const logger = async (req, res, next) => {
+    const currentDate = new Date().toJSON().slice(0, 10).replace(/-/g, '/');
+    let logs = await readFile("./server/data/log.json")
+    if (req.body.item !== undefined) {
+        logs = [...logs, `Item: ${req.body.item}, Date : ${currentDate}`]
+        writeToFile("./server/data/log.json", logs)
```

Comment:
should put the following inside the fileHelper but you should just use:
`fs.appendFile('message.txt', 'data to append', (err) => { ... });`